logger.go

```diff
@@ -129,8 +132,13 @@ func Reset() {
 	// if there's no backends at all configured, we could use some tricks to
 	// automatically setup backends based if we have a TTY or not.
 	sequenceNo = 0
-	b := SetBackend(NewLogBackend(os.Stderr, "", log.LstdFlags))
-	b.SetLevel(DEBUG, "")
+	if runtime.GOOS == "windows" {
+		b := SetBackend(NewLogBackend(colorable.NewColorableStderr(), "", log.LstdFlags))
+		b.SetLevel(DEBUG, "")
+	} else {
+		b := SetBackend(NewLogBackend(os.Stderr, "", log.LstdFlags))
+		b.SetLevel(DEBUG, "")
+	}
```

Comment:
You don't need to separate with platforms. On UNIX, colorable.NewColorableStderr() return os.Stderr.

Reply:
OK